esp/templates/users/profiles/usercontact.html

```diff
@@ -1,5 +1,11 @@
 <!-- User ContactInfo -->
 
+<style>
+#id_receive_txt_message {
```

Comment:
Can you put this in a proper css file?  I think forms.css is probably already included in the relevant pages.